oxidation/crates/poc/src/lib.rs

```diff
@@ -0,0 +1,31 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BSLv1.1 (eventually AGPLv3) 2016-2021 Scille SAS
+
+mod parser;
+mod utils;
+
+use proc_macro::TokenStream;
+use std::fs::File;
+use std::io::Read;
+use syn::{parse_macro_input, LitStr};
+
+use crate::utils::quote_struct;
+
+#[proc_macro]
+pub fn parsec_schema(path: TokenStream) -> TokenStream {
+    let path = parse_macro_input!(path as LitStr);
+    let file_path = std::env::current_dir().unwrap().join(&path.value());
```

Comment:
I there an official doc stating that `std::env::current_dir()`  always point to the crate source directory when running a proc_macro ? (and if yes, we should add a comment here ^^)
This is typically the kind code that breaks when compiling from another crate...